src/masternodes/govvariables/attributes.cpp

```diff
@@ -282,6 +302,20 @@ Res ATTRIBUTES::ProcessVariable(const std::string& key, const std::string& value
 
     auto typeKey = itype->second;
 
+    if (type == AttributeTypes::Param) {
+        if (typeId == ParamIDs::DFIP2201) {
+            if (typeKey == DFIPKeys::RewardPct ||
+                typeKey == DFIPKeys::BlockPeriod) {
+                return Res::Err("Unsupported type for DFIP2201 {%d}", typeKey);
+            }
+        } else {
```

Comment:
It might not change current functionality...

I would have expected an `else if` which checks `typeId` against `ParamIDs::DFIP2203`, in order to avoid unexpected early returns.